jbi/services/bugzilla.py

```diff
@@ -142,7 +142,10 @@ def check_health(self) -> ServiceHealth:
         all_webhooks_enabled = False
         if logged_in:
             jbi_webhooks = self.client.list_webhooks()
-            all_webhooks_enabled = len(jbi_webhooks) > 0
+            if len(jbi_webhooks) == 0:
+                logger.info("No webhooks enabled")
```

Comment:
I'm a fan of returning as early as possible because then I know the rest of this function doesn't apply if these conditions are met. Does this read better to you if we return a health object immediately if `len(jbi_webhooks) == 0`? It would also remove the else statement.

Reply:
Added [1cf9906](https://github.com/mozilla/jira-bugzilla-integration/pull/783/commits/1cf9906c813d34a75e8f54aba29f263d596bf2e3) and [6b6098a](https://github.com/mozilla/jira-bugzilla-integration/pull/783/commits/6b6098acd978186add2876ab7d2e596768c4c1b1) (and tagged you for a re-review). Lmk what you think

Reply:
Oh yeah that's really clean. I like that